system-settings-plugin/online-accounts.settings

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Accounts",
-    "icon": "preferences-desktop-accounts-symbolic",
+    "icon": "account",
```

Comment:
The old name is correct, per the Icon Naming Spec. If there these two icons are different in suru, then probably we should fix the theme, not change the names here.

Reply:
ah, cool. so make sure the file it references is the same, got it :)